wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Schema.old.kt

```diff
@@ -13,8 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/*
 package com.squareup.wire.schema
 
 expect class Schema internal constructor(protoFiles: Iterable<ProtoFile>) {
   val protoFiles: List<ProtoFile>
 }
+*/
```

Comment:
Done this way to preserve the history of the `Schema.kt` file. Will push a quick follow-up to fix the names.